expected-consensus.md

```diff
@@ -295,8 +295,9 @@ Due to the existence of potential forks, a miner can try to unduly influence pro
 This is detectable when a miner submits two blocks that satisfy either of the following "slashing conditions":
 
 (1) one block contains at least one ticket in its ticket array generated at the same round as one of the tickets in the other block's ticket array.
-(2) one block's parent is a tipset that does not, but could validly, include the other block.
-  - While it cannot be proven that a miner omits known blocks from a TipSet in general (i.e. network latency could simply mean the miner did not receive a particular block) in this case it can be proven because a miner must be are aware of a block they mined in a previous round.
+(2) one block's ancestor TipSet does not include the other block when it should have.
```

Comment:
Ah I misunderstood that this holds for all ancestors instead of just the parent, good catch.

However if you replace ancestor for parent in my comment I still prefer the modification I pushed.  While the comment here uses simpler language and is less dense I believe that it is less exact and meaningful to an implementer of these rules than the words I used.  I will not block the PR but please consider that "should" is laden with assumptions that are not stated.

Reply:
Maybe a better phrasing is "one block's ancestor is a tipset that could have validly included the other block according to tipset validity rules, however that ancestor of the first block does not include the other block"

Reply:
Ok works for me. What do you think of just adding "(in spite of the fact that both were mined by the same miner)" at the end.

If you don't have the fact that the two blocks are from the same miner (which you plainly state up top) cached in, this reads as a much broader condition: i.e. any forgotten block means slash, which makes the below confusing.

Feel free to ignore. Maybe I'm too careful here.

Reply:
This is a genuinely interesting discussion to have about writing style.  My strong belief is that attempting to write no more than what is necessary actually helps clarify communication in contexts like this where readers are looking for precision and paying close attention.  I definitely tend more towards this than a lot of people and maybe too much so.

Fighting for style decisions doesn't feel exactly in scope for me yet (though I hope it will soon as we converge on correctness).  I trust your sense of the spec's style as a whole so by all means add the extra reminder if you feel inspired.

Reply:
no. I agree with you. I have the inverse problem, and i believe synthesis is always better. Having no proof that this is more/less clear to anyone but myself. Leaving as is.

Thanks for engaging earnestly with this.